azure-app-service-vnet-template/Infrastructure/azure_sql.bicep

```diff
@@ -0,0 +1,43 @@
+param sql_server_name string
+param sql_db_name string
+param location string
+param sku_name string
+param sku_tier string
+param vnet_subnet_id string
+param tags object
+
+resource sql_server 'Microsoft.Sql/servers@2022-02-01-preview' = {
+  name: sql_server_name
+  location: location
+  properties: {
+    administratorLogin: 'sqlsrvdbadmin'
```

Comment:
make the username and password parameterized

Reply:
Resolved